From b29270233ddaa9f00ca662a1f6c823b15f93a5b4 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 13 Sep 2020 13:34:44 +0300
Subject: [PATCH] [SW-17444] preboot: goya2_arc: increase fdt regions

Since goya2 has zephyr images along with uboot image to be verified,
the max fdt regions set to 10 won't suffice. This patch increases max
fdt regions to 20.

Change-Id: I5b02fb06c8c8f7c8335f7ecf44ae927c3d415444
---

diff --git a/preboot/fit.c b/preboot/fit.c
index 647d5dc..67f1ba3 100644
--- a/preboot/fit.c
+++ b/preboot/fit.c
@@ -34,7 +34,7 @@
 #define FIT_ALGO_PROP_EXPECTED_VALUE	"sha256,rsa3072"
 
 #define FIT_MAX_PATH		200
-#define MAX_FDT_REGIONS		10
+#define MAX_FDT_REGIONS		20
 #define IMAGE_MAX_HASHED_NODES	20
 
 struct fdt_region fdt_regions[MAX_FDT_REGIONS];
